core/controlplane/config/templates/cloud-config-controller

```diff
@@ -261,9 +263,10 @@ coreos:
         --register-with-taints=node.alpha.kubernetes.io/role=master:NoSchedule \
         --allow-privileged=true \
         --pod-manifest-path=/etc/kubernetes/manifests \
-        --cluster-dns={{.DNSServiceIP}} \
-        --cluster-domain=cluster.local \
-        --cloud-provider=aws \
+        {{ if .KubeDns.NodeLocalResolver }}--cluster-dns=${COREOS_PRIVATE_IPV4} \
+        {{ else }}--cluster-dns={{.DNSServiceIP}} \
+        {{ end }}--cluster-domain=cluster.local \
+        --cloud-provider=aws
```

Comment:
@redbaron some merge must have gone wrong here - the `\` has been removed so latest master updates fail right now.